moveit_core/online_signal_smoothing/src/butterworth_filter.cpp

```diff
@@ -100,12 +100,12 @@ bool ButterworthFilterPlugin::initialize(rclcpp::Node::SharedPtr node, moveit::c
 {
   node_ = node;
   num_joints_ = num_joints;
+  param_listener_ = std::make_shared<online_signal_smoothing::ParamListener>(node_);
```

Comment:
Since `param_listener_` is only used in this one place, it doesn't need to be a member variable. Just make it a local variable here.